rero_ils/modules/holdings/jsonschemas/holdings/holding-v0.0.1.json

```diff
@@ -257,7 +257,7 @@
           "type": "string",
           "format": "date",
           "title": "The expected date for the next issue",
-          "pattern": "\\d{4}-((0[1-9])|(1[0-2]))-(((0[1-9])|[1-2][0-9])|(3[0-1]))$",
+          "pattern": "^[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[01])$",
           "validationMessage": "Should be in the following format: 2022-12-31 (YYYY-MM-DD).",
```

Comment:
Maybe its be interesting to add a validation message to resources which do not have one